runtime/support.py

```diff
@@ -309,7 +365,25 @@ def setup(import_roots, zip_safe):
 
     # Extract files to disk if necessary
     if not zip_safe:
-        extract_dir = _extract_files(archive_path)
+        extract_only = os.environ.get("UNPAR_EXTRACT_ONLY")
```

Comment:
Maybe we should drop the `UN` prefix... I know we've been calling this feature unpar, but both in the manifest and in the environment variables it might make sense to just have it be `PAR_` prefix

Reply:
Yeah, particularly for the environment variable it doesn't make as much sense. The manifest file I think can be whatever we want, but I suppose you're right in that it's really a manifest of what's in the par, not what's in the unpar'd directory.